autogen/oai/client.py

```diff
@@ -158,8 +123,19 @@ def _client(self, config: Dict[str, Any], openai_config: Dict[str, Any]) -> Open
         after removing extra kwargs.
         """
         openai_config = {**openai_config, **{k: v for k, v in config.items() if k in self.openai_kwargs}}
-        self._process_for_azure(openai_config, config)
-        client = OpenAI(**openai_config)
+        api_type = config.get("api_type")
```

Comment:
```suggestion
        api_type = openai_config.get("api_type")
```
same for below.

Reply:
Why do you suggest to use `openai_config`? It will never have `api_type` since it is a custom attribute not present in `OpenAI.__init` and will be sorted out

<img width="912" alt="image" src="https://github.com/microsoft/autogen/assets/7947027/1350ee6e-6d00-4483-ab0c-b6e2a6f56e08">

Reply:
You're right. My bad.